geometry_stamped_publisher:
  ros__parameters:

    wait_sec_between_publish: 6

    goal_names: ["pos1", "pos2", "pos3", "pos4"]
    pos1:
      position: [0.12, 0.29, 0.61]
      orientation: [-0.5, 0.5, 0.5, 0.5 ]
    pos2: 
      position: [0.45, 0.29, 0.61]
      orientation: [ -0.5, 0.5, 0.5, 0.5 ]
    pos3:
      position: [0.45, 0.61, 0.16]
      orientation: [ -0.5, 0.5, 0.5, 0.5 ]
    pos4:
      position: [0.17  , 0.61, 0.55]
      orientation: [ -0.5, 0.5, 0.5, 0.5 ]

